### PR TITLE
added tryCatch() block to read_wikitables

### PR DESCRIPTION
**Pull request by niannucci**

### CODE REVIEW

File: R/read_table.R [hunk 1]
```diff
@@ -44,6 +44,18 @@ read_wikinodes <- function(url, replace_linebreak = ", ", ...) {
 
 read_wikitables <- function(url, ...) {
   read_wikinodes(url) %>%
-    rvest::html_table(fill = TRUE)
+  tryCatch(
```
Comment (beanumber) [inline R/read_table.R]: Whoa, will that work with a pipe like that?

Comment (beanumber) [reply R/read_table.R]: Please indent

File: R/read_table.R [hunk 2]
```diff
@@ -44,6 +44,18 @@ read_wikinodes <- function(url, replace_linebreak = ", ", ...) {
...
+    expr = {
+      rvest::html_table(fill = TRUE)
+    },
+    error = function(e){
+      message('Caught an error!')
```
Comment (beanumber) [inline R/read_table.R]: Can we give a more informative error message? *Why* didn't it work? In the Marvel example the problem is that there is a cell that spans multiple rows, so we might say something about that.

File: R/read_table.R [hunk 3]
```diff
@@ -44,6 +44,18 @@ read_wikinodes <- function(url, replace_linebreak = ", ", ...) {
...
+      print(e)
+    },
+    warning = function(w){
+      message('Caught a warning!')
+      print(w)
+    }
+  )
 }
```
Comment (beanumber) [inline R/read_table.R]: Can you please add a corresponding test?

File: DESCRIPTION [hunk 4]
```diff
@@ -9,10 +9,7 @@ Authors@R: c(
   person("Benjamin S.", "Baumer", role = "ctb", 
            comment = c(ORCID = "0000-0002-3279-0516"))
   )
-Description: A collection of functions to read in and clean tables from Wikipedia. It allows any Wikipedia table to be 
-read into R so that it can then be used as a typical R dataframe and manipulated as such. It also cleans up the dataframes 
-so that they are able to be used with tidyverse functions. This package allows for many more datasets to be used in R so that
-users can explore data on any subject they desire, while cleaning the dataframes to avoid issues.
+Description: A collection of functions to read in and clean tables from Wikipedia. It allows any Wikipedia table to be read into R so that it can then be used as a typical R dataframe and manipulated as such. It also cleans up the dataframes so that they are able to be used with tidyverse functions. This package allows for many more datasets to be used in R so that users can explore data on any subject they desire, while cleaning the dataframes to avoid issues.
```
Comment (beanumber) [inline DESCRIPTION]: Actually, these are supposed to be broken into lines of at most 80 characters, so the other way is better. Can you fix this?

**Review by beanumber — CHANGES_REQUESTED** [review]

File: R/read_table.R (hunk 2 above)
Comment (niannucci) [reply R/read_table.R]: would this be the only case that returns an error here, or do I need to do an if statement? I was trying to do an if to see if statement to see if the error messaged contained a a string from what this returns, but I'm having issues with the error being an invalid regular expression and not working with grepl

**Pull request merged**